assets/src/edit-story/components/dropzone/dropZoneProvider.js

```diff
@@ -38,6 +39,9 @@ function DropZoneProvider({ children }) {
   const [dropZones, setDropZones] = useState([]);
   const [hoveredDropZone, setHoveredDropZone] = useState(null);
 
+  const [isDragging, setIsDragging] = useState(false);
```

Comment:
Q: Curious, why use `isDragging` instead of using the found `hoveredDropZone`?

The logic in `onDragOver` is detecting which dropzone is hovered on while dragging and sets the relevant dropzone as `hoveredDropZone` -- this state can be used within the dropzone to detect if that specific dropzone is being hovered over.

Reply:
I had issues with drag over and sub elements, removing the drag state. If you have an idea how this could work better, an example PR would be useful for me to see this here.

Reply:
Sounds good, I could create a sample PR, will try to get to it today, perhaps you're right and some issues come out.